### PR TITLE
Adding more enums

### PR DESCRIPTION
**Pull request by chenss3**
Fixes: #1069 
Fixes: #1079
Fixes: #1078 
Fixes: #1077 
Fixes: #1122
Fixes: #1239

Metadata diff:
```
# NIIF FLAGS
Windows.Win32.UI.Shell.Apis.NIIF_ERROR removed
Windows.Win32.UI.Shell.Apis.NIIF_ICON_MASK removed
Windows.Win32.UI.Shell.Apis.NIIF_INFO removed
Windows.Win32.UI.Shell.Apis.NIIF_LARGE_ICON removed
Windows.Win32.UI.Shell.Apis.NIIF_NONE removed
Windows.Win32.UI.Shell.Apis.NIIF_NOSOUND removed
Windows.Win32.UI.Shell.Apis.NIIF_RESPECT_QUIET_TIME removed
Windows.Win32.UI.Shell.Apis.NIIF_USER removed
Windows.Win32.UI.Shell.Apis.NIIF_WARNING removed
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_ERROR added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_ICON_MASK added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_INFO added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_LARGE_ICON added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_NONE added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_NOSOUND added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_RESPECT_QUIET_TIME added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_USER added
Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS.NIIF_WARNING added
Windows.Win32.UI.Shell.NOTIFYICONDATAA.dwInfoFlags(X64, Arm64)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS
Windows.Win32.UI.Shell.NOTIFYICONDATAA.dwInfoFlags(X86)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS
Windows.Win32.UI.Shell.NOTIFYICONDATAW.dwInfoFlags(X64, Arm64)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS
Windows.Win32.UI.Shell.NOTIFYICONDATAW.dwInfoFlags(X86)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_INFOTIP_FLAGS
# NIS 
Windows.Win32.UI.Shell.Apis.NIS_HIDDEN removed
Windows.Win32.UI.Shell.Apis.NIS_SHAREDICON removed
Windows.Win32.UI.Shell.NOTIFY_ICON_STATE added
Windows.Win32.UI.Shell.NOTIFY_ICON_STATE.NIS_HIDDEN added
Windows.Win32.UI.Shell.NOTIFY_ICON_STATE.NIS_SHAREDICON added
Windows.Win32.UI.Shell.NOTIFYICONDATAA.dwState(X64, Arm64)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_STATE
Windows.Win32.UI.Shell.NOTIFYICONDATAA.dwState(X86)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_STATE
Windows.Win32.UI.Shell.NOTIFYICONDATAW.dwState(X64, Arm64)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_STATE
Windows.Win32.UI.Shell.NOTIFYICONDATAW.dwState(X86)...System.UInt32 => Windows.Win32.UI.Shell.NOTIFY_ICON_STATE
# SHOW_WIND_CMD
Windows.Win32.UI.Shell.Apis.ShellExecuteA : nShowCmd...Int32 => SHOW_WINDOW_CMD
Windows.Win32.UI.Shell.Apis.ShellExecuteW : nShowCmd...Int32 => SHOW_WINDOW_CMD
# KNOWN_FOLDER_FLAG
Windows.Win32.UI.Shell.Apis.SHGetKnownFolderPath : dwFlags...UInt32 => KNOWN_FOLDER_FLAG
# SYS_COLOR_INDEX
Windows.Win32.Graphics.Gdi.Apis.GetSysColorBrush : nIndex...Int32 => SYS_COLOR_INDEX
Windows.Win32.UI.WindowsAndMessaging.Apis.GetSysColor => Windows.Win32.Graphics.Gdi.Apis.GetSysColor
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_3DDKSHADOW removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_3DFACE removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_3DHIGHLIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_3DHILIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_3DLIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_3DSHADOW removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_ACTIVEBORDER removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_ACTIVECAPTION removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_APPWORKSPACE removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_BACKGROUND removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_BTNFACE removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_BTNHIGHLIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_BTNHILIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_BTNSHADOW removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_BTNTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_CAPTIONTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_DESKTOP removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_GRADIENTACTIVECAPTION removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_GRADIENTINACTIVECAPTION removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_GRAYTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_HIGHLIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_HIGHLIGHTTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_HOTLIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_INACTIVEBORDER removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_INACTIVECAPTION removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_INACTIVECAPTIONTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_INFOBK removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_INFOTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_MENU removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_MENUBAR removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_MENUHILIGHT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_MENUTEXT removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_SCROLLBAR removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_WINDOW removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_WINDOWFRAME removed
Windows.Win32.UI.WindowsAndMessaging.SYS_COLOR_INDEX.COLOR_WINDOWTEXT removed
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_3DDKSHADOW added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_3DFACE added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_3DHIGHLIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_3DHILIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_3DLIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_3DSHADOW added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_ACTIVEBORDER added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_ACTIVECAPTION added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_APPWORKSPACE added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_BACKGROUND added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_BTNFACE added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_BTNHIGHLIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_BTNHILIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_BTNSHADOW added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_BTNTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_CAPTIONTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_DESKTOP added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_GRADIENTACTIVECAPTION added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_GRADIENTINACTIVECAPTION added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_GRAYTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_HIGHLIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_HIGHLIGHTTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_HOTLIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_INACTIVEBORDER added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_INACTIVECAPTION added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_INACTIVECAPTIONTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_INFOBK added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_INFOTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_MENU added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_MENUBAR added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_MENUHILIGHT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_MENUTEXT added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_SCROLLBAR added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_WINDOW added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_WINDOWFRAME added
Windows.Win32.Graphics.Gdi.SYS_COLOR_INDEX.COLOR_WINDOWTEXT added
# PANOSE 
Windows.Win32.System.SystemServices.PAN_ARM_STYLE.PAN_ARM_ANY added
Windows.Win32.System.SystemServices.PAN_ARM_STYLE.PAN_ARM_NO_FIT added
Windows.Win32.System.SystemServices.PAN_CONTRAST.PAN_CONTRAST_ANY added
Windows.Win32.System.SystemServices.PAN_CONTRAST.PAN_CONTRAST_NO_FIT added
[Omitted for length]
Windows.Win32.System.SystemServices.PAN_XHEIGHT.PAN_XHEIGHT_DUCKING_LARGE added
Windows.Win32.System.SystemServices.PAN_XHEIGHT.PAN_XHEIGHT_DUCKING_SMALL added
Windows.Win32.System.SystemServices.PAN_XHEIGHT.PAN_XHEIGHT_DUCKING_STD added
Windows.Win32.System.SystemServices.PAN_XHEIGHT.PAN_XHEIGHT_INDEX added
```

### CODE REVIEW

File: generation/WinSDK/enums.json [hunk 1]
```diff
@@ -31894,6 +31898,224 @@
...
+      "filter": "PAN_PROP_",
+      "header": "wingdi.h"
+    },
+    "members": [
+      {
+        "name": "PAN_PROP_ANY",
+        "value": 0
+      },
+      {
+        "name": "PAN__PROP_NO_FIT",
```
Comment (mikebattista) [inline generation/WinSDK/enums.json:32024]: Remove extra `_`.

File: generation/WinSDK/enums.json [hunk 2]
```diff
@@ -31894,6 +31898,224 @@
...
+  },
+  {
+    "name": "PAN_LETTER_FORM",
+    "autoPopulate": {
+      "filter": "PAN_LETT_",
+      "header": "wingdi.h"
+    },
+    "members": [
+      {
+        "name": "PAN_LETTER_FORM_ANY",
```
Comment (mikebattista) [inline generation/WinSDK/enums.json:32074]: PAN_LETT to match the other member names in the enum.

File: generation/WinSDK/enums.json [hunk 3]
```diff
@@ -31894,6 +31898,224 @@
...
+        "value": 0
+      },
+      {
+        "name": "PAN_LETTER_FORM_NO_FIT",
```
Comment (mikebattista) [inline generation/WinSDK/enums.json:32078]: Same as above.

**Review by mikebattista — CHANGES_REQUESTED** [review]

**Comment by elachlan** [comment]
@chenss3 can you please apply the changes from #1243 into this pr?

File: generation/WinSDK/enums.json [hunk 4]
```diff
@@ -14898,6 +14898,10 @@
       {
         "method": "GetSysColor",
         "parameter": "nIndex"
+      },
+      {
+        "method": "GetSysColorBrush",
+        "parameter": "nIndex"
       }
```
Comment (elachlan) [inline generation/WinSDK/enums.json:14905]: Add missing usage:
```json
{
        "interface": "ITextHost",
        "method": "TxGetSysColor",
        "parameter": "nIndex"
      }
```

File: generation/WinSDK/enums.json [hunk 5]
```diff
@@ -14898,6 +14898,10 @@
       {
```
Comment (elachlan) [inline generation/WinSDK/enums.json:14898]: Change `type` to int and use `autopopulate`:
```
"type": "int",
    "autoPopulate": {
      "filter": "COLOR_",
      "header": "WinUser.h"
    },
    "members": [],
```

File: generation/WinSDK/enums.json [hunk 6]
```diff
@@ -31894,6 +31763,224 @@
...
+      {
+        "name": "PAN_FAMILY_ANY",
+        "value": 0
+      },
+      {
+        "name": "PAN_FAMILY_NO_FIT",
+        "value": 1
+      }
+    ],
+    "uses": []
```
Comment (elachlan) [inline generation/WinSDK/enums.json:31821]: All of the PAN enums are missing uses. Is this on purpose?

**Comment by elachlan** [comment]
Add Fixes: #1239 to link to the issue for closure.

File: generation/WinSDK/enums.json (hunk 6 above)
Comment (chenss3) [reply generation/WinSDK/enums.json]: Not on purpose! Just added them!

**Review by mikebattista — APPROVED** [review]

**Pull request merged**